### PR TITLE
PAAS-1026: clean-up repository, update README

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-1026

Short description:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
 These scripts fulfil various maintenance tasks.
 
-## Apply monitoring tagging
-Setups up a cron entry in order to retrieve tags from various sources (AWS API, filesystem files, environment variables) and set it in the datadog.yaml file in the tag entry.
+## jahia-cloud-env-update
+
+These manifests are run when a new Jahia cloud version is released, they aim at updating environments to make them aligned with modifications on universal/unomi install JPS and Docker images.
+
+The whole process is fully documented here: https://confluence.jahia.com/display/PAAS/Jahia+Cloud+release+process+-+Ref+ISSOP08.A14210
+
```
Comment (laurentfufu) [inline README.md:12]: I don't know if this link is a good idea

File: README.md [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+## Archives
+
+### backrest & backup-restore
+
+These are (very) old versions of backup & restore scripts/manifests which are now maintained in paas_jelastic_backup repository.
+
+### disable-phpmyadmin
+
+This manifest's purpose was to disable phpmyadmin on database nodes by stopping httpd & removing the init script. It is now handled by install.yaml manifest of paas_jelastic_dx_universal repository.
+
```
Comment (laurentfufu) [inline README.md:34]:
```suggestion
Manifest's purpose was to disable phpmyadmin on database nodes by stopping httpd & removing the init script. It is now handled by install.yaml manifest of paas_jelastic_dx_universal repository.

```

File: README.md (hunk 1 above)
Comment (kevinvandel) [reply README.md:12]: +1 (a fortiori on a public repo)

Comment (adoux) [reply README.md]: Done!

**Pull request merged**